vss-extension.json

```diff
@@ -76,23 +76,29 @@
                 "inputDescriptors": [
                     {
                         "id": "datadog_site",
-                        "isRequired": true,
                         "name": "Datadog Site",
                         "description": "Datadog Site to send data to",
-                        "valueHint": "datadoghq.com",
-                        "inputMode": "textbox"
+                        "inputMode": "textbox",
+                        "values": {
+                            "defaultValue": "datadoghq.com"
+                        },
+                        "validation": {
+                            "dataType": "string",
+                            "isRequired": true
+                        }
                     },
                     {
                         "id": "api_key",
                         "isRequired": true,
                         "name": "Datadog API Key",
                         "description": "Datadog API Key",
                         "inputMode": "passwordBox",
+                        "isConfidential": true,
```

Comment:
I don't know what this does but it's on the spec and I guess it cannot harm just in case

Reply:
I guess it doesn't display it